src/pg/sql/getArgSql.js

```diff
@@ -73,10 +73,20 @@ export default function getArgSql(
     getJsonBuildTrusted({ $order: sql`${JSON.stringify($order)}::jsonb` });
 
   const cursorKey = getJsonBuildTrusted({
-    $cursor: sql`jsonb_build_array(${join(groupCols || orderCols)})`,
+    $cursor:
+      $group === true
+        ? sql`jsonb_build_array('')`
```

Comment:
Interesting, was:

```
sql`''`;
```
not working? I did not think Graffy assumes the cursor is always an array, but I might be wrong.

Reply:
Yes, this works. Modified it locally. Going to commit. 🙏